main.go

```diff
@@ -18,8 +19,17 @@ func main() {
 		out.Warn("%s", err.Error())
 	}
 
+	var jiraRepo *jira.Repository
+	if c.JIRAIntegration != (config.JIRAConfig{}) {
+		jiraRepo = jira.New(jira.RepositoryConfig{
+			AuthToken:   c.JIRAIntegration.APIToken,
+			Email:       c.JIRAIntegration.UserEmail,
+			JIRAAddress: c.JIRAIntegration.Host,
+		})
+	}
+
 	filesystem := fs.New(c)
-	timetrace := core.New(c, filesystem)
+	timetrace := core.New(c, filesystem, []core.Provider{jiraRepo})
```

Comment:
I'm not actually sure this is correct on review; if the jiraRepo is nil, it likely shouldn't be passed into the core at all.